gcc-arm.cmake

```diff
@@ -0,0 +1,13 @@
+# CMake Toolchain File to cross-compile rawrtc for arm using gcc
+
+#target system is a linux
+set (CMAKE_SYSTEM_NAME Linux)
+#target processor is an arm with a version number <8
+#this is relevant since the only problematic part of rawrtc is the intel-specific assembler code in crc32c.c and arm supports its own assembler instructions since version 8
```

Comment:
This line should only be needed in case the redirect tool is being compiled, correct? In that case, it should probably be if-def'ed, too.

Reply:
If I get this working the way I plan to, the only lines that will be needed are the ones that set CC and CXX. That can be done just as well via environment variables, so I'll (hopefully) delete `gcc-arm.cmake`.